ToolBox/RandomDataGenerator/RandomDataGeneratorProgram.cs

```diff
@@ -58,7 +58,7 @@ string dividendEveryQuarterPercentageString
                 var coarseFiles = CoarseUniverseGenerator.CoarseUniverseGeneratorProgram.ProcessEquityDirectories(
                     Globals.DataFolder,
                     false
-                ).ToList();
```

Comment:
Removed this call, as on next line returned object is iterated. Which will make ToList call unnecessary